e2e/common/common.go

```diff
@@ -35,28 +46,67 @@ func NewConfig(
 	validators, account, numEthAccounts int,
 	useFnConsensus bool,
 ) (*lib.Config, error) {
-	basedirAbs, err := filepath.Abs(path.Join(BaseDir, name))
+	checkAppHashOnExitEV := os.Getenv(checkAppHashOnExitEv)
+	checkAppHashOnExit := len(checkAppHashOnExitEV) > 0
+
+	LoomPath := os.Getenv(loomExeEv)
+	if len(LoomPath) == 0 {
+		LoomPath = DefaultLoomPath
+	}
+
+	LoomPath2 := os.Getenv(loomExe2Ev)
+	if len(LoomPath2) == 0 {
+		LoomPath2 = DefatulLoomPath2
+	}
+
+	var err error
+	validatorsEv := os.Getenv(ValidatorsEv)
+	if len(validatorsEv) > 0 {
+		validators, err = strconv.Atoi(validatorsEv)
```

Comment:
I don't think we should override the number of validators like this because it breaks some of the tests that rely on the number of validators (like chainconfig voting test). We should check if the number of validators in the test and the number of validators from env match before overriding it. 

For example, suppose `ValidatorsEv` is 2, and `AltValidatorsEv` is 2
Case1: a test requires 4 validators, it should works fine.
Case2: a test requires 1 validator, it should keep the number of validator to be 1.
Case3: a test requires 6 validators, 4 of them will use loomExeEv and 2 of them will use loomExe2Ev

Then we should gradually fix our test not to run with a single validator.